### ISSUE TITLE
<Create consistency in how claims are labeled "Submitted on" vs. "Received"> from <Benefits Management Tools>

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#72494]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Benefits Management Tools`
- **OCTO product owner:** `Zach Goldfine`
- **Product name:** `Claim Status Tool`
- **Product manager:**  @jacobworrell 
- **Slack channel:** `#benefits-management-tools`
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** @davidakennedy 
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Currently in the Claim Status tool we describe claims as "Submitted on" a certain date. The problem is, claims can be submitted by the Veteran on a certain date but not actually initiated by the VA until some days later (e.g. physical claims take a while to get delivered via mail, or claims submitted with BDD aren't established until Veteran separates from service).

So this string in some cases might show a very different date than when the Veteran thinks of the claim as having been "submitted," which could lead to confusion or distrust.

In the VA mobile app, claims are described as "Received on" and "Submitted on" interchangeably (see screenshots in Supporting Artifacts).

The date shown reflects the "Claim date" that appears in VBMS, the service VSRs use to develop claims. @jerekshoe and I are doing some research to figure out _exactly_ what this "Claim date" refers to so we can create the most accurate user-facing string. 

The goal of this ticket is to track that work at a broader sitewide level, so that our findings can be coordinated with other teams (e.g. VA Mobile app team) that currently use this date or might use this date in the future.

#### CST

<img width="504" alt="Screenshot 2023-12-15 at 12 15 38 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/1244eb6b-72fa-4fe6-9fb2-25e799dd13d5"> <br>

<img width="504" alt="Screenshot 2023-12-15 at 12 16 03 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/0e48fe57-cb19-4365-9b5a-be4098790275">

#### VA Mobile App

![IMG_7510](https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/45b2b2c4-d7a3-482d-a70d-9b2287529cd4) <br>

![IMG_7511](https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/b4c79a57-5da1-48df-95e4-9be37a9fe686)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Enter your description here`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by coforma-terry** [comment]
Nothing actionable from CAIA at this time.

**Comment by skylerschain** [comment]
Just want to provide an update here that [per this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72065) we have changed the string from "Submitted on" to "Received on" in CST.
